### PR TITLE
Add readme for all services

### PR DESCRIPTION
**Pull request by VathsalaAchar**

### CODE REVIEW

**Comment by VathsalaAchar** [comment]
One check for the analytics service has failed here but has been fixed in [this PR](https://github.com/VathsalaAchar/MLA-pilot-group4/pull/8)

I'll only merge this PR after the bug fix is merged in.

**Review by LakshmiAvanthiKaneti — APPROVED** [review]
That's a great work @VathsalaAchar  , seems like run pytest has a failure because of no tests but then obviously you pulled another PR fixing that.

**Comment by VathsalaAchar** [comment]
Thanks! That's just it - it was unexpected to have pytest fail in the build but fixed now.

**Pull request merged**
